src/components/MarkdownImage.tsx

```diff
@@ -0,0 +1,41 @@
+import Image from "next/image"
+import { Flex } from "@chakra-ui/react"
+import clamp from "lodash.clamp"
+
+import { CONTENT_IMAGES_MAX_WIDTH } from "@/lib/constants"
+
+interface MarkdownImageProps {
+  src: string
```

Comment:
This will use always the default img `src`, we would need to update the way we compute this path for translated images later after setting i18n

Reply:
Agreed. Later with i18n in place, the `src` will come with the locale in the path but, as you said, we can take care of that logic in `getStaticProps`.